core/services/bulletprooftxmanager/eth_broadcaster.go

```diff
@@ -120,7 +120,7 @@ func (eb *ethBroadcaster) monitorEthTxs() {
 	for {
 		pollDBTimer := time.NewTimer(databasePollInterval)
 
-		keys, err := eb.store.Keys()
+		keys, err := eb.store.SendingKeys()
```

Comment:
Did you also check EthConfirmer?

Might be worth grepping the whole codebase for uses of `Keys`.

Or, even better, remove `Keys()` method entirely and create two different methods to replace it - `AllKeys` and `SendKeys`. This seems safest.

Reply:
I did exactly that: https://github.com/smartcontractkit/chainlink/pull/3402/files#diff-1cae1a151fc8ab6377e1472c0aed45faR95 The eth_confirmer change is below

I like the `AllKeys()`, I'll add that